composer.json

```diff
@@ -22,16 +22,18 @@
         "sylius/theme-bundle": "^2.0"
     },
     "require-dev": {
+        "friendsofphp/php-cs-fixer": "^2.17",
         "jackalope/jackalope-doctrine-dbal": "^1.3",
-        "symfony/dotenv": "^4.4 || ^5.0",
-        "phpunit/phpunit": "^8.5.9",
         "jangregor/phpstan-prophecy": "^0.8",
         "phpstan/phpstan": "^0.12",
         "phpstan/phpstan-doctrine": "^0.12",
         "phpstan/phpstan-phpunit": "^0.12",
         "phpstan/phpstan-symfony": "^0.12",
-        "thecodingmachine/phpstan-strict-rules": "^0.12",
-        "symfony/phpunit-bridge": "^5.1"
+        "phpunit/phpunit": "^8.5.9",
+        "symfony/dotenv": "^4.4 || ^5.0",
+        "symfony/monolog-bundle": "^3.6",
```

Comment:
without this, a lot of deprecations are logged to the console when executing the lint script

Reply:
@nnatter I would really like to avoid unneeded requirement. Think we need to find another way or just live with the deprecation messages. PS: 3.6 would also be the false version we support ^3.1 in sulu/sulu.

Reply:
but maybe we should add errors to file would make it easier to debug 🤔

Reply:
as far as i see, we have this as dependency for other bundles too. but dont have a strong opinion 🙂

Reply:
for all I didn't yet cleanup the dependency chaos. there I think still depending on ffmpeg 🙈

Reply:
but as said monolog is maybe a good idea to have.